config/locales/client.en.yml

```
@@ -6911,11 +6911,10 @@ en:
           auto_update: "Auto update when Discourse is updated"
           color_scheme: "Color Palette"
           dark_color_scheme: "Dark Color Palette"
-          edit_color_scheme: "Edit Color Palette"
-          edit_dark_color_scheme: "Edit DarkWColor Palette"
           default_light_scheme: "Light (default)"
-          color_scheme_select: "Select colors to be used by theme"
-          dark_color_scheme_select: "Select colors to be used by theme in dark mode"
+          color_scheme_select: "Select colors to be used by theme."
+          dark_color_scheme_select: "Select colors to be used by theme in dark mode."
+          edit_colors: "Edit colors."
```

```suggestion
          edit_colors: "Edit colors"
```

an action link probably shouldn't end with a period?